### ISSUE TITLE
Volume slider

### ISSUE THREAD

**Issue by Defreezer** [FreeTubeApp/FreeTube#1054]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
One particular item that's always peeved me for many years is the size of the volume slider on many different control UIs for videos, including Youtube's. I've always found them too small to find a pleasant volume level without tinkering with the volume mixer of the program itself.

**Describe the solution you'd like to see implemented**
Would it be possible to extend the volume slider a little bit on the player's UI for a more accurate control of the volume level? It has plenty of space for it, I reckon. Alternatively, what if the volume slider was able to change the volume of the program itself on the mixer, like Audacity does?

**Screenshots**
![1](https://user-images.githubusercontent.com/79490923/108793222-cc9c3980-7561-11eb-9736-6eb9a722f0f3.png)

**Comment by Murciegalo** [comment]
I could put an icon ( same as youtube) to make easier selecting it.
Can take this issue if administrators assign to me
Thanks

**Comment by Defreezer** [comment]
You mean that little round dot on Youtube's volume slider? I actually find Freetube to be better without it, the only real problem is that it's a tiny slider.

**Comment by GilgusMaximus** [comment]
@Murciegalo Thanks for the offer, but as @Defreezer mentioned, I am not too sure whether that solved the problem. Because the audio bar only has so many ticks for a given size. So we need to increase the number of ticks available, via extending the size of it in order to fit more ticks.

In regards to your alternative idea, this would only work, if Electron has an abstraction for that feature. Otherwise it will be the wild west with all the different Linux Operating systems out there. So it would be better to have a solution that is independent of the operating system.

**Comment by PAI5REECHO** [comment]
Came here and searched the issues to see if someone had brought this up before and I'm pleasantly glad and surprised it's not just me that finds the size of volume sliders on every video player ridiculously small.
![image](https://user-images.githubusercontent.com/96688573/159137840-b4b8e9a4-75ce-4e0c-a2e9-2d0368e1995a.png)
There's so much free space. It should be 3 times the width it currently is and allow for the granularity of controlling the volume to half of a percent intervals

**Comment by Defreezer** [comment]
Small update on this: In case increasing ticks on the volume slider proves to be too much of a burden, I've found an alternative method that might be more efficient.

The browser extension "Enhancer for Youtube" has a small function for the volume slide where, when active, the user can change the volume of the video by holding the right mouse button, then scrolling the mouse wheel up or down. This function also allows the user to decide the rate of the volume change (ex.: one tick of the mouse wheel may raise or lower the volume by 1 point for maximum precision). I've found this quite an elegant solution.

![volume](https://user-images.githubusercontent.com/79490923/205303416-09b7d9b4-87f1-4ed0-b640-7a6e835a9532.png)
